manager/controllers/app/modules/moduleinterface.go

```diff
@@ -113,14 +114,14 @@ func (m *Selector) SupportsGovernanceActions(module *app.FybrikModule, actions [
 }
 
 // SupportsGovernanceAction checks whether the module supports the required governance action
-func (m *Selector) SupportsGovernanceAction(module *app.FybrikModule, action *pb.EnforcementAction) bool {
+func (m *Selector) SupportsGovernanceAction(module *app.FybrikModule, action *openapiclientmodels.ResultItem) bool {
 	// Check if the module supports the capability
 	if hasCapability, caps := utils.GetModuleCapabilities(module, m.Capability); hasCapability {
 		// There could be multiple structures for the same CapabilityType
 		for _, cap := range caps {
 			// Loop over the data transforms (actions) performed by the module for this capability
 			for _, act := range cap.Actions {
-				if act.ID == action.Id && act.Level == action.Level {
+				if act.ID == (action.GetAction().Name + "-ID") {
```

Comment:
Why do we need this `-ID` thing?

Reply:
Earlier manager was comparing actions with action ID. In the current taxonomy changes, we donot provide provision for action id. There is only action name. We may need to change manager test case configurations to remove dependency on ID. @shlomitk1 can you suggest what could be the changes involved? I can do that in this PR itself if it is not too complex. I just want to cross verify once with you before I do the changes. Thanks.

Reply:
@roee88 @simanadler Based on discussion with @shlomitk1 , I have updated the mock policy manager and unit test configurations to avoid concatenation with "-ID" string. The hardcoding is now removed. 

However, I think there should be a separate issue (as @shlomitk1 suggested) to update the [ModuleCapability struct](https://github.com/rohithdv/fybrik/blob/247b95cf32adc1d712c75305f01ffc1c9ab3ef52/manager/apis/app/v1alpha1/fybrikmodule_types.go#L149). The `SupportedAction`  class used in `ModuleCapability struct` should be replaced by the [action taxonomy model struct](https://github.com/rohithdv/fybrik/blob/taxonomy_manager_changes/pkg/taxonomy/model/base/model_action.go). This is the larger issue which deals with how the action taxonomy will be used by the manager and the modules. Hence it would not be in the scope of this PR in my opinion.